.github/workflows/tests.yml

```diff
@@ -133,4 +133,4 @@ jobs:
           source "${HOME}/.bash_env"
           nim --version
           echo
-          nimble --verbose test
+          nimble --verbose test --hints:off
```

Comment:
This also suppresses useful hints (ie XDeclaredButNotUsed)
See https://github.com/status-im/nim-libp2p/blob/master/config.nims

Reply:
Thanks!